SC:
  SC_sim_single:
    ros__parameters:
      velocity: 12
      steering_delay: 0 # 1 step = 0.01s (at 100 hz steering)
      navsat_noise_std: 1e-6 # standard deviation of simulated GPS noise (in degrees lat/long)
      pose: 'Hill1_SC'

  SC_controller:
    ros__parameters:
      dist: 0.0
      traj_name: 'buggycourse_safe.json'
      controller: 'stanley'
